controllers/object_controls.go

```diff
@@ -1213,6 +1265,36 @@ func TransformToolkit(obj *appsv1.DaemonSet, config *gpuv1.ClusterPolicySpec, n
 			}
 		}
 	}
+
+	// Update the driver-root volume if a custom driver install directory is configured with the operator
```

Comment:
Having the driver transform logic in a separate function would help to make this more testable.

Reply:
Done.